src/frontend/src/components/SearchPage/SearchResults.jsx

```diff
@@ -54,7 +58,11 @@ const SearchResults = ({ text, filter }) => {
 
   return (
     <Container className={classes.searchResults}>
-      {data && data.length ? <Timeline pages={[data]} /> : <h1>No results search</h1>}
+      {data && data.length ? (
+        <Timeline pages={data} nextPage={() => setSize(size + 1)} />
```

Comment:
I guess if ES is 0-based, we don't need the `+ 1`.

Reply:
oh I thought the size + 1 was to increment the index no?

Reply:
btw I changed the page query range limit from 0-999 because of this.